app/src/main/java/org/glucosio/android/db/DatabaseHandler.java

```diff
@@ -750,30 +746,28 @@ public void editPressureReading(long oldId, PressureReading reading) {
         return idArray;
     }
 
-    public ArrayList<Integer> getMinPressureReadingAsArray() {
+    public List<Double> getMinPressureReadingAsArray() {
```

Comment:
The method name does not reflect the return type.

Reply:
maybe `getMinPressureReadingAsList`?

Reply:
Also renamed